libr/cons/cons.c

```diff
@@ -1677,7 +1677,11 @@ R_API void r_cons_set_last_interactive(void) {
 }
 
 R_API void r_cons_set_title(const char *str) {
+#if __WINDOWS__
+	SetConsoleTitle (str);
```

Comment:
You have to use "wide" version of this API and you need to convert the input string. Actually it displays garbage (treats str as `wchar_t*`:
![image](https://user-images.githubusercontent.com/14978853/92767056-6e0e4500-f39f-11ea-8222-8f9af0244413.png)

Reply:
I suggest it should look like:
```
#if __WINDOWS__
#  if defined(_UNICODE)
wchar_t* wstr = r_utf8_to_utf16_l(str, strlen(str));
if (wstr) {
    SetConsoleTitleW (wstr);
    R_FREE(wstr);
}
#  else // defined(_UNICODE)
SetConsoleTitle (str);
# endif
#endif // __WINDOWS__
```

Reply:
applied your patch

Reply:
I have compiled your recent commit with the following meson command to turn on the unicode support:
`python sys\meson.py --shared --install --options "c_args=-D_UNICODE -DUNICODE"`

and it works well with utf8 console charset in default Windows terminal:
![image](https://user-images.githubusercontent.com/14978853/92981671-967f6780-f4a3-11ea-82b1-8a7fb46e5977.png)